### PR TITLE
Update delete functionality

### PR DESCRIPTION
**Pull request by Theophileaseh**
**In this project , I:**
Add a column species of type string to the animals table.
Inserted the following data:

- Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
- Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
- Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
- Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
- Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
- Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
- Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.

Updated the animals table inside a transaction by setting the species column to unspecified. Rolled back the change and verified that the species columns went back to the state before the transaction.
Inside a transaction:
Updated the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Updated the animals table by setting the species column to pokemon for all animals that don't have species already set.
Committed the transaction.
Verified that change was made and persists after commit.
[Screenshot](https://github.com/Theophileaseh/Vet-clinic-database/blob/b74de9db6acc451872a25659c75866cd203b3b82/s1.1.png)

Delete all records in the animals table, then rolled back the transaction.
Verified if all records in the animals table still exist.
[Screenshot](https://github.com/Theophileaseh/Vet-clinic-database/blob/b74de9db6acc451872a25659c75866cd203b3b82/s1.2.png)

In a transaction, I:

- Deleted all animals born after Jan 1st, 2022.
- Created a savepoint for the transaction.
- Updated all animals' weight to be their weight multiplied by -1.
- Rollback to the savepoint
- Updated all animals' weights that are negative to be their weight multiplied by -1.
- Committed transaction

[Screenshot](https://github.com/Theophileaseh/Vet-clinic-database/blob/b74de9db6acc451872a25659c75866cd203b3b82/s2.png)

Wrote queries to answer the following questions:

- How many animals are there?
- How many animals have never tried to escape?
- What is the average weight of animals?
- Who escapes the most, neutered or not neutered animals?
- What is the minimum and maximum weight of each type of animal?
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?

[Screenshot](https://github.com/Theophileaseh/Vet-clinic-database/blob/b74de9db6acc451872a25659c75866cd203b3b82/s4.png)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+COMMIT;
+
+BEGIN;
+DELETE * from animals WHERE date_of_birth > '01-01-2022';
+
+UPDATE animals SET weight_kg = weight_kg * -1;
```
Comment (Mre55) [inline data.sql]: Please move all of these queries to `queries.sql` file.

Comment (Mre55) [inline data.sql]: We don't need to multiply all of animal weight by -1 that way the positive weights will be negative, we only need to multiply the ones which are negative. Kindly, fix that.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @Theophileaseh,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
Hi @Theophileaseh
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**